index.html

```diff
@@ -0,0 +1,131 @@
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+        </section> 
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+        </section>
+        <section class="item"></section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+        </section>
+        <section class="item"></section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
```

Comment:
- [ ] Please add the pseudo classes for all the buttons(active, hover and pressed)